src/midi/MidiMessage.cpp

```diff
@@ -16,6 +16,10 @@
 #include <vector>
 #include <iostream>
 
+#ifdef _WIN32
+#include <iterator>
```

Comment:
Would it be okay to leave it here until @craigsapp modifies his repo? I would take care of removing it once the midifile lib has been changed.